Cargo.toml

```diff
@@ -117,11 +130,16 @@ tokio-stream = "0.1.14"
 tower = "0.4.13"
 tower-http = "0.3.5"
 tracing = "0.1.35"
+tracing-opentelemetry = "0.22.0"
 tracing-subscriber = "0.3.16"
-txn_forwarder = {path = "tools/txn_forwarder"}
+txn_forwarder = { path = "tools/txn_forwarder" }
 url = "2.3.1"
+vergen = "8.2.1"
 wasi = "0.7.0"
 wasm-bindgen = "0.2.83"
+yellowstone-grpc-client = { git = "https://github.com/rpcpool/yellowstone-grpc.git", tag = "v1.14.1+solana.1.17.28" }                          # tag is geyser plugin
+yellowstone-grpc-proto = { git = "https://github.com/rpcpool/yellowstone-grpc.git", tag = "v1.14.1+solana.1.17.28" }                           # tag is geyser plugin
+yellowstone-grpc-tools = { git = "https://github.com/rpcpool/yellowstone-grpc.git", tag = "v1.14.1+solana.1.17.28", default-features = false } # tag is geyser plugin
```

Comment:
I'm fine with this but are you holding off on publishing the yellowstone crates?

Reply:
Let me sync with the team on this about publishing to crates.io

Reply:
https://lib.rs/crates/yellowstone-grpc-client

NVM